HelenasBank/Classes/BankHelper.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace HelenasBank.Classes
+{
+    public class BankHelper
+    {
+        public decimal PerformDeposit(decimal balance, int ammount)
```

Comment:
Skulle nog vara bättre med double eller decimal :)

Reply:
Gällande insättning och uttag så tänkte jag att i en vanlig bankomat så får man ju inte ta ut "exakta" summor så körde på lite samma princip. :)

Reply:
Men skadar såklart inte att göra om till decimal så allt är av samma datatyp. :)

Reply:
Now my dear friend I havve changed to decimal :)